HW8/MariiaDiachuk/08_Practical_Task_Structure_Project/utils/__init.py

```
@@ -0,0 +1,3 @@
+from .formatter import *
```

Ruff (0.8.2)

`from .formatter import *` used; unable to detect undefined names

(F403)